website/docs/configuration.md

```diff
@@ -114,6 +114,19 @@ let g:neovide_profiler = v:false
 Setting this to `v:true` enables the profiler, which shows a frametime graph in the upper left
 corner.
 
+#### Underline automatic scaling
+
+```vim
+let g:neovide_underline_automatic_scaling = v:true
```

Comment:
Note that the config docs should document the default, and not one possible configuration.